rfcs/build-system/05-folder-organization.md

```diff
@@ -0,0 +1,63 @@
+# RFC: Fluent UI Repo Folder Organization
+
+---
+
+Contributors: Fluent UI React Build team, @justSlone
+
+## Summary
+
+This propsoal covers 3 topics:
+1. organization of rfcs folder
+2. organization (or removal) of specs folder
+3. organization of packages folder (can be made later)
```

Comment:
I think at the moment we should stay focused and cover only first 2 points to address immediate issue, then follow up by updating this RFC to cover point no.3. Also may speed up merging this etc.   WDYT?

Reply:
Yeah, I agree. I'll move packages folder to out of scope. I wanted to start with it in to get feedback and comments on it, and understand the desire to do the larger change. Looks like there is clear desire to move forward with it, but I think there is value in moving on rfcs/specs first.